pkg/controllers/deprovisioning/metrics.go

```diff
@@ -50,8 +50,8 @@ var (
 		prometheus.HistogramOpts{
 			Namespace: metrics.Namespace,
 			Subsystem: deprovisioningSubsystem,
-			Name:      "replacement_machine_initialized_seconds",
-			Help:      "Amount of time required for a replacement machine to become initialized.",
+			Name:      "replacement_node_initialized_seconds",
```

Comment:
Along with this and all other metrics changes.

Reply:
Yes it does. We need to add this with your taint changes so that we package all the breaking changes together.